### PR TITLE
chore: add more utils to docker images

### PR DESCRIPTION
**Pull request by douglaz**
Closes https://github.com/fedimint/fedimint/issues/5954

Busybox is very complete. curl and rsync may be also be util.

### CODE REVIEW

File: nix/flakebox.nix [hunk 1]
```diff
@@ -610,6 +610,9 @@ rec {
           pkgs.coreutils
           pkgs.sqlite
           pkgs.fakeNss
+          pkgs.busybox
```
Comment (oleonardolima) [inline nix/flakebox.nix:613]: I wonder if using `busybox` pkg can remove the need for `fakeNss`, need to take a look at it.

**Comment by m1sterc001guy** [comment]
What does `busybox` add?

**Comment by oleonardolima** [comment]
> What does `busybox` add?

@m1sterc001guy It adds a bunch of UNIX utilities (check below), but @douglaz do we need all of that ?

> I know from experience that we only find out we need it when most need it in debugging, so I might be biased. 😅

image

**Comment by douglaz** [comment]
> > What does `busybox` add?
> 
> @m1sterc001guy It adds a bunch of UNIX utilities (check below), but @douglaz do we need all of that ?

It provide many useful commands but it's also very lightweight:
```
 ls -lh /nix/store/6723i29mnr4gib2qp5vahfr7xv7h8ad8-busybox-1.36.1/bin/busybox
-r-xr-xr-x 1 root root 1.2M Jan  1  1970 /nix/store/6723i29mnr4gib2qp5vahfr7xv7h8ad8-busybox-1.36.1/bin/busybox
```
So there is no reason to not add it.

**Review by m1sterc001guy — APPROVED** [review]

**Comment by oleonardolima** [comment]
> It provide many useful commands but it's also very lightweight:
> 
> ```
>  ls -lh /nix/store/6723i29mnr4gib2qp5vahfr7xv7h8ad8-busybox-1.36.1/bin/busybox
> -r-xr-xr-x 1 root root 1.2M Jan  1  1970 /nix/store/6723i29mnr4gib2qp5vahfr7xv7h8ad8-busybox-1.36.1/bin/busybox
> ```
> 
> So there is no reason to not add it.

Oh, got it. Thanks!

**Pull request merged**